File: php/program/Shirt.php
<?php
// import file
require ('Clothing.php');

// deklarasi kelas Shirt yang merupakan anak dari Clothing
class Shirt extends Clothing {
    // atribut private
    private $color = '';
    private $sleeve_type = '';
    private $foto = '';

    // construktor
    public function __construct( $color = '', $sleeve_type = '', $foto = '' ) {
        $this->color = $color;
        $this->sleeve_type = $sleeve_type;
        $this->foto = $foto;
    }
    // getter dan setter
    public function get_color(){
        return $this->color;
    }
    public function set_color($color){
        $this->color = $color;
    }
    
    public function get_sleeve_type(){
        return $this->sleeve_type;
    }
    public function set_sleeve_type($sleeve_type){
        $this->sleeve_type = $sleeve_type;
    }

    public function get_foto(){
        return $this->foto;
    }
    public function set_foto($foto){
        $this->foto = $foto;
    }
}

?>

File: python/program/Shirt.py
from Clothing import Clothing

# deklarasi class

# deklarasi class menjadi anak dari Clothing


class Shirt(Clothing):

    # deklarasi private atribut
    __color = ""
    __sleeve_type = ""

    # constructor

    def __init__(self, idProduct="", nama="", brand="", price="", color="", sleeve_type="", size="", material="", gender=""):
        super().__init__(idProduct, nama, brand, price, size, material, gender)
        self.__color = color
        self.__sleeve_type = sleeve_type

    # setter dan getter
    def get_color(self):
        return self.__color

    def set_color(self, color):
        self.__color = color

    def get_sleeve_type(self):
        return self.__sleeve_type

    def set_sleeve_type(self, sleeve_type):
        self.__sleeve_type = sleeve_type
